tests/framework/data/FilterTest.php

```diff
@@ -15,21 +18,25 @@ public function testBuildCondition()
         $value_array  = [15, 16];
         $filter       = new Filter();
 
-        $condition = $filter->buildCondition('>=', $field_name, $value_int);
-        $this->assertEquals($condition, ['>=', $field_name, $value_int]);
-        $condition = $filter->buildCondition('<=', $field_name, $value_int);
-        $this->assertEquals($condition, ['<=', $field_name, $value_int]);
-        $condition = $filter->buildCondition('>', $field_name, $value_int);
-        $this->assertEquals($condition, ['>', $field_name, $value_int]);
-        $condition = $filter->buildCondition('<', $field_name, $value_int);
-        $this->assertEquals($condition, ['<', $field_name, $value_int]);
-        $condition = $filter->buildCondition('!=', $field_name, $value_int);
-        $this->assertEquals($condition, ['NOT', [$field_name => $value_int]]);
-        $condition = $filter->buildCondition('!=', $field_name, $value_array);
-        $this->assertEquals($condition, ['NOT IN', $field_name, $value_array]);
-        $condition = $filter->buildCondition(null, $field_name, $value_int);
-        $this->assertEquals($condition, [$field_name => $value_int]);
-        $condition = $filter->buildCondition('like', $field_name, $value_string);
-        $this->assertEquals($condition, ['like', $field_name, $value_string, false]);
+        $this->assertEquals($filter->buildCondition('>=', $field_name, $value_int), ['>=', $field_name, $value_int]);
```

Comment:
`assertEquals` signature is `assertEquals($expected, $actual)`. Flip the arguments please